examples/kernel_shap_wine_lr.ipynb

```diff
@@ -0,0 +1,920 @@
+{
```

Comment:
<p>Rogue cell?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/192/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>Hmmm, that was definitely removed at some point ..</p>